src/test/java/com/commercetools/sync/categories/CategorySyncTest.java

```diff
@@ -45,6 +48,23 @@
     private List<String> errorCallBackMessages;
     private List<Throwable> errorCallBackExceptions;
 
+    // protected method access helper
+    private static class CategorySyncTestie extends CategorySync {
```

Comment:
I would just name it `CategorySyncMock` :)

Reply:
Right (just a habit overtaken from some project), yours sound much better, changed :)